.travis.yml

```diff
@@ -1,38 +1,44 @@
 language: php
+os: linux
+dist: xenial
 
-sudo: false
-
-dist: trusty
+install:
+  - travis_retry composer require phpunit/phpunit:^7.5
 
-matrix:
+jobs:
   include:
-   - php: 5.3 # Composer requires PHP 5.3.2+ to run, so we cannot test below 5.3
+   - php: 5.3 # Composer and PHPUnit require PHP 5.3.2+ to run, so we cannot test below 5.3
      dist: precise # PHP 5.3 is supported only on Precise.
+     install: travis_retry composer require phpunit/phpunit:^4.8
```

Comment:
This is unnecessary, as composer would automatically select best matching version.
See https://github.com/smarty-php/smarty/blob/master/composer.json#L43

Reply:
If you actually want to change something to ensure version choices, change `config.platform`. The rest should be done automatically.

Reply:
Thank you for your input! I agree your suggestion is better. I remember running into issues with travis builds failing because it was picking the wrong phpunit version. But I'm not sure why, so I'll try again.

Reply:
Well, that worked beautifully. Not sure what went wrong earlier. Thanks again!

Reply:
Ordering matters. If first specified version matches current constraints, Composer won't try the next one.